app/assets/stylesheets/_colours.scss

```diff
@@ -0,0 +1,78 @@
+$black:         #000;
```

Comment:
@klepas um. q. How do I pull in the compiled css? is this the stuff in /assets of ui-kit?